OMEdit/OMEditLIB/Element/Element.cpp

```diff
@@ -2337,7 +2317,7 @@ void Element::createStateElement()
     extents << QPointF(-100, -100) << QPointF(100, 100);
     mpStateElementRectangle->setExtents(extents);
   } else {
-    mpStateElementRectangle = 0;
+    mpStateElementRectangle.release();
```

Comment:
We should call `reset` here instead of `release`.

Reply:
Yes, good catch.